samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -80,6 +86,15 @@ public long getOffsetFileTTL() {
    *         on the job status server.
    */
   public List<String> getJobStatusServers() {
-     return Arrays.asList(StringUtils.split(get(CONFIG_JOB_STATUS_SERVERS), '.'));
+     return Arrays.asList(StringUtils.split(get(CONFIG_JOB_STATUS_SERVERS), ','));
+  }
+
+  /**
+   *
```

Comment:
I think our current checkstyle settings require javadoc for public methods. I'd personally prefer skipping them when they're redundant/obvious as well. In any case, if we add them let's make them consistent.